{"uid":"101296cc5464eaed","name":"Тест: создание курьера с существующим логином возвращает ошибку","fullName":"tests.test_create_courier.TestCreateCourier#test_create_courier_with_existing_login_returns_error","historyId":"51fdc006abe2cdd0ed8c7b1570ec5b92","time":{"start":1758283131186,"stop":1758283133980,"duration":2794},"status":"passed","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[],"afterStages":[],"labels":[{"name":"feature","value":"Создание курьера"},{"name":"parentSuite","value":"tests"},{"name":"suite","value":"test_create_courier"},{"name":"subSuite","value":"TestCreateCourier"},{"name":"host","value":"DESKTOP-26M7FOD"},{"name":"thread","value":"1220-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_create_courier"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[],"tags":[]},"source":"101296cc5464eaed.json","parameterValues":[]}